modules/mcuboot/CMakeLists.txt

```diff
@@ -2,6 +2,12 @@ add_subdirectory(${ZEPHYR_MCUBOOT_MODULE_DIR}/boot/bootutil/zephyr
                  ${CMAKE_CURRENT_BINARY_DIR}/boot/bootutil/zephyr
 )
 
+if(SYSBUILD)
+  # Code below is partition manager w/ child image support, thus return when
+  # sysbuild is used.
+  return()
+endif()
+
```

Comment:
There is some vital code underneath this, which doesn't appear to be ported over yet.

For now, I think you can safely move this check below the `if (CONFIG_MCUBOOT)` block, so that we can at least support generating the `ram_flash` region for multi-image update.

Reply:
that code needs to be rw-written cause the sysbuild handling doesn't support the `set_shared()` feature as it is being used.

So this will need to be reworked and is not as simple as moving the check below.

Reply:
Ok. Moving the check down worked for me locally, but it's okay to do this later.

Reply:
yes, the call themselves will not fail, but `set_shared()` will not be sharing anything to sysbuild.